microsoft/testsuites/core/cpu.py

```diff
@@ -19,6 +27,47 @@
     """,
 )
 class CPU(testsuite.TestSuite):
+    @testsuite.TestCaseMetadata(
+        description="""
+            This test will check that CPU assigned to lsvmbus
+            channels cannot be put offline.
+            Steps :
+            1. Get the list of lsvmbus channel cpu mappings using
+            command `lsvmbus -vv`.
+            2. Create a set of cpu's assigned to lsvmbus channels.
+            3. Try to put cpu offline by running
+            `echo 0 > /sys/devices/system/cpu/cpu/<cpu_id>/online`.
+            Note : We skip cpu 0 as it handles system interrupts.
+            4. Ensure that cpu is still online by checking state '1' in
+            `/sys/devices/system/cpu/cpu/<target_cpu>/online`.
+            """,
+        priority=1,
+    )
+    def cpu_verify_online(self, node: Node, log: Logger) -> None:
```

Comment:
the case name can be more specified. like `cpu_verify_vmbus_force_online`? Other name is also ok.

Reply:
Sure! This name conveys the intent better.